content/src/content/jcr_root/apps/core/wcm/components/form/text/v2/text/clientlibs/site/js/text.js

```diff
@@ -88,10 +95,19 @@
                 event.target.setCustomValidity(this._properties.requiredMessage);
             }
         }
+        if (displayValidationMessage) {
+            let validationMessage = event.target.parentElement.querySelector(".cmp-form-text__validation-message");
```

Comment:
@LSantha  This can be problematic. I just pulled in all the changes from master in my branch to create a private build and it fails in CM with error - 

```
Errors:
--> JSC_LANGUAGE_FEATURE. This language feature is only supported for ECMASCRIPT6 mode or better: let declaration. at libs/core/wcm/components/form/text/v2/text/clientlibs/site.min.js line 99 : 12source: libs/core/wcm/components/form/text/v2/text/clientlibs/site.min.js line: 99
```

You can fix this by either not using let or by adding a jsProcessor for ECMASCRIPT support like [0]

[0] https://github.com/adobe/aem-core-wcm-components/commit/034884c1b85719c380648cb61ecb49ce5acb348a

Reply:
@mohiaror , I've opened this PR to fix this issue https://github.com/adobe/aem-core-wcm-components/pull/2924